pipeline/update_nodedb.sh

```diff
@@ -109,35 +114,36 @@ fi
 
 if [[ "${copy}" == true ]]; then
   echo "-- Copying files to nodedb for ${whoami} --"
+  if [[ ! -d ${logs}/${whoami,,} ]]; then
+    echo "[ERROR] Log path does not exist"
+    exit
+  fi
   # Assuming deploy log directory link is lowercase of WHOAMI
-  cp ${logs}/${whoami,,}/shard?.txt ${nodedb}/${target} -v
+  cp ${logs}/${whoami,,}/shard?.txt ${nodedb}/${target}
+  cp -r ${logs}/${whoami,,}/init ${nodedb}/${target}
 fi
 
 # Push to master on nodedb
 if [[ "${push}" == true ]]; then
+  pushd ${nodedb}
+  git add ${target}/*
   if [[ "${yes}" == true ]]; then
     echo "-- Pushing nodedb update --"
   else
+    git status
     read -rp "Push nodedb update? [Y/N]" reply
     echo
     if [[ "${reply}" != "Y" ]]; then
       exit
+      popd
     fi
   fi
-  pushd ${nodedb}
-  git add ${target}/*
-  git commit -m "[update_watchdog] Updating ip lists for ${target}"
+  git commit -m "[update_nodedb] Updating ip lists & init files for ${target}"
```

Comment:
do you need to make a new commit each time?  if it is not updated?  you need to check if any change.

Reply:
Added check for changes in nodedb, before committing & pushing
Also updated [`testnet_nodedb.py`](https://github.com/harmony-one/nodedb/pull/17) to keep the order of IPs as returned from `aws-instances.sh`